soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3956,6 +3982,14 @@ void DrawRandoEditor(bool& open) {
                                             "expected to be collected after getting Sun's Song.");
                         PaddedSeparator();
 
+                        // Shuffle Cows
+                        ImGui::Text(Settings::ShuffleCows.GetName().c_str());
+                        InsertHelpHoverText(
+                            "Enabling this will let cows give you items upon performing Epona's Song in "
+                            "front of them. There are 9 cows.");
```

Comment:
Open to opinions here, but I don't think we need to specify how many cows are available, especially since there is an extra one in Master Quest which will be relevant someday.